ewatercycle/config/_validators.py

```diff
@@ -138,4 +138,6 @@ def validate_path(value, allow_none=False):
     # wflow specific
     'wflow.singularity_image': validate_string_or_none,
     'wflow.docker_image': validate_string_or_none,
+    'pcrglobwb.singularity_image': validate_string_or_none,
```

Comment:
the default ewatercycle.yaml file (also in this package) does not contain the new keys, by default I get a "key error".

Reply:
Good point! Done

Reply:
Thanks!